modular_skyrat/modules/self_actualization_device/code/self_actualization_device.dm

```diff
@@ -35,12 +35,11 @@
 	var/next_fact = 10
 	/// A list containing advertisements that the machine says while working.
 	var/static/list/advertisements = list(\
-	"Thank you for using the Self-Actualization Device, brought to you by Cinco: A Family Company!", \
+	"Thank you for using the Self-Actualization Device, brought to you by Veymed, because you asked for it.", \
 	"The Self-Actualization device is not to be used by the elderly without direct adult supervision. Cinco is not liable for any and all injuries sustained under unsupervised usage of the Self-Actualization Device.", \
 	"Please make sure to clean the Self-Actualization Device every fifteen minutes! The Self-Actualization Device is not to be used un-cleaned.", \
-	"Before using the Self-Actualization Device, remove your teeth. You don't want your pearly whites getting in the way of the Self-Actualization Device!" , \
-	"Please make sure to have your pre-Self Actualization Device brain-stimulating full-body gel lube session performed by a licensed lube man.", \
```

Comment:
I kind of like the lube man line idk

Reply:
It was removed to make all the lines vaguely around the same tier, and because a machine telling you to get your full-body lubing from a lubeman isn't exactly a good look if we're trying to aim for 'more serious.'

Reply:
good enough